modules/contact-form/admin.php

```diff
@@ -37,7 +37,7 @@ function grunion_admin_css() {
 	if ( is_null( $current_screen ) ) {
 		return;
 	}
-	if ( ! in_array( $current_screen->id, array( 'edit-feedback', 'jetpack_page_omnisearch', 'dashboard_page_omnisearch' ) ) ) {
+	if ( ! in_array( $current_screen->id, array( 'edit-feedback' ) ) ) {
```

Comment:
I think php's internals optimize to this internally anyway, but this way is a simpler diff and more extensible down the road if we add more pages on later again.

I'm not sold on its necessity, but no real harm in swapping back.  Tweak as you like! <3